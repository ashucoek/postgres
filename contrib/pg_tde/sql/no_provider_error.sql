CREATE EXTENSION pg_tde;

-- should fail
CREATE TABLE t1 (n INT) USING tde_heap;

-- should work
CREATE TABLE t2 (n INT) USING heap;

DROP TABLE t2;

DROP EXTENSION pg_tde;
