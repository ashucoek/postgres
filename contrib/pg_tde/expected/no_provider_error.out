CREATE EXTENSION pg_tde;
-- should fail
CREATE TABLE t1 (n INT) USING tde_heap;
ERROR:  failed to retrieve principal key. Create one using pg_tde_set_principal_key before using encrypted tables.
-- should work
CREATE TABLE t2 (n INT) USING heap;
DROP TABLE t2;
DROP EXTENSION pg_tde;
